sphinxcontrib/confluencebuilder/directives.py

```diff
@@ -57,6 +58,23 @@ def run(self):
         self.state.nested_parse(self.content, self.content_offset, node)
         return [node]
 
+class ConfluenceExcerptDirective(Directive):
+    has_content = True
+    option_spec = {
+        'name': directives.unchanged,
+    }
+
+    def run(self):
+        self.assert_has_content()
+        text = '\n'.join(self.content)
+
+        node = confluence_excerpt(rawsource=text)
+        if 'name' in self.options:
+            node['name'] = self.options['name']
```

Comment:
And if the `hidden` option is added, maybe this section should be tweaked to something such as:

```
        for k, v in self.options.items():
            params[kebab_case_to_camel_case(k)] = v
```